docroot/themes/contrib/civictheme/civictheme_library/components/01-atoms/select/select.scss

```diff
@@ -8,9 +8,14 @@
   @include civictheme-typography('text-regular');
 
   display: block;
-  font-weight: 600;
   margin: 0 auto 0.5rem;
-  padding: civictheme-space(1.5) civictheme-space(3) civictheme-space(1.5) civictheme-space(2);
+  padding: civictheme-space(1.25) civictheme-space(2);
+  border-radius: $civictheme-select-border-radius;
+
+  &.disabled {
```

Comment:
@joshua-salsadigital 
Disabled state should be done through the `disabled` attribute.

Could you please update twig to inject `disabled` attribute into `atrributes` array if `is_disabled` prop is `true`